decayAmplitude/waveDescription.cc

```diff
@@ -582,9 +732,33 @@ waveDescription::waveNameFromTopology(isobarDecayTopology         topo,
 			if (vertex->massDependence() && vertex->massDependence()->name() == "binned") {
 				const particle& P = *(currentVertex->daughter2());
 				binnedMassDependencePtr massDep = static_pointer_cast<binnedMassDependence>(vertex->massDependence());
-				waveName << "[" << spinQn(P.isospin()) << parityQn(P.G()) << ","
+				waveName << "binned[" << spinQn(P.isospin()) << parityQn(P.G()) << ","
 				         << spinQn(P.J()) << parityQn(P.P()) << parityQn(P.C()) << ","
 				         << massDep->getMassMin() << "," << massDep->getMassMax() << "]";
```

Comment:
Would it not make sense to do the printing of all the quantum numbers only once, after determining the name of the mass dependence function?